modules/nextflow/src/main/resources/nextflow/trace/assets/ReportTemplate.js

```diff
@@ -166,6 +166,7 @@ $(function() {
     if (bytes == '-' || bytes == 0){
       return bytes;
     }
+    bytes = norm_mem([bytes]);
```

Comment:
Does this change work? `bytes` is expected to be a number instead `norm_mem` returns an array.